site/en/external/lockfile.md

```diff
@@ -224,3 +224,47 @@ practices:
 By following these best practices, you can effectively utilize the lockfile
 feature in Bazel, leading to more efficient, reliable, and collaborative
 software development workflows.
+
+## Merge Conflicts {:#merge-conflicts}
+
+The lockfile format is designed to minimize merge conflicts, but they can still
+happen.
+
+### Automatic Resolution {:#automatic-resolution}
+
+Bazel provides a custom
+[git merge driver](https://git-scm.com/docs/gitattributes#_defining_a_custom_merge_driver)
+to help resolve these conflicts automatically.
+
+Set up the driver by adding this line to a `.gitattributes` file in the root of
+your git repository:
+
+```gitattributes
+MODULE.bazel.lock merge=bazel-lockfile-merge
+```
+
+Then each developer who wants to use the driver has to register it once by
+following these steps:
+
+1. Install [jq](https://jqlang.github.io/jq/download/) (1.5 or higher).
+2. Run the following commands:
+
+```bash
+jq_script=$(curl https://raw.githubusercontent.com/bazelbuild/bazel/release-7.2.0/scripts/bazel-lockfile-merge.jq)
```

Comment:
I bet no one is going to maintain this line by updating for new releases, any ideas how to avoid that?

Reply:
The script only needs to change if we make a major change to the lockfile format, not on every bump to `lockFileVersion`, so this should be very rare and e.g. Bazel 7.3.0 would just reuse this link. 

@Wyverald Is there any way to stamp "current Bazel version" into the docs? If not, could this be covered by `// LINT.IfChange`?

Reply:
@Wyverald Friendly ping.

I replaced this with a reference to `master` for now. If we can guarantee backwards compatibility indefinitely (for as long as `MODULE.bazel.lock` exists), that would be ideal as we can then register the script globally. Versioning it per project and Bazel version would be tedious anyway.